### PR TITLE
Changes to Command instructions pages for post-refactoring changes

### PR DESCRIPTION
**Pull request by theboomer**
Changes to various Command pages to reflect new import packages post-Refactoring; modified example code to compile, and changed page description on CommandServices now being CommandManager

### CODE REVIEW

**Comment by theboomer** [comment]
I did not touch the very last page on the low-level command handling cause I dont understand it, couldn't make a corresponding matched compiliable example...

File: source/plugin/commands/arguments.rst [hunk 1]
```diff
@@ -174,25 +174,27 @@ The parser in this example reads two input arguments and converts them to a vect
 .. code-block:: java
 
    import com.flowpowered.math.vector.Vector2i;
-   import org.spongepowered.api.util.command.args.ArgumentParseException;
-   import org.spongepowered.api.util.command.args.CommandArgs;
+   import org.spongepowered.api.command.args.ArgumentParseException;
+   import org.spongepowered.api.command.args.CommandArgs;
    import org.spongepowered.api.text.Text;
-   import org.spongepowered.api.util.command.args.CommandElement;
+   import org.spongepowered.api.command.args.CommandElement;
```
Comment (ryantheleach) [inline source/plugin/commands/arguments.rst]: Not sure if they have been pruned to match style guides for the docs, but this code-block is missing import statements required for the example to compile.

```

import com.flowpowered.math.vector.Vector2i;
import org.spongepowered.api.command.CommandSource;
import org.spongepowered.api.command.args.ArgumentParseException;
import org.spongepowered.api.command.args.CommandArgs;
import org.spongepowered.api.command.args.CommandContext;
import org.spongepowered.api.command.args.CommandElement;
import org.spongepowered.api.text.Text;
import org.spongepowered.api.text.Texts;

import java.util.Collections;
import java.util.List;
```

**Comment by ryantheleach** [comment]
Other then that, the changes compile. may be other issues on the page, only tested the examples that had significant code changes rather then just imports changing.

File: source/plugin/commands/arguments.rst (hunk 1 above)
Comment (Inscrutable) [inline source/plugin/commands/arguments.rst]: **[Docs Guideline 14](https://docs.spongepowered.org/en/contributing/spongedocs.html)**: *Imports should be written out in code blocks the first time they are referenced in each article, but not repeated after the first time*.

Comment (Minecrell) [inline source/plugin/commands/arguments.rst]: The trailing whitespace added here isn't needed though.

**Comment by Tzky** [comment]
closed with d77cfcf

**Pull request closed without merging**